client/my-sites/post-share/style.scss

```diff
@@ -172,3 +174,170 @@ $section-border: solid 1px transparentize( lighten( $gray, 20% ), .5 );
 	}
 }
 
+.post-share__footer-nav {
+	margin: 0;
+
+}
+
+.post-share__footer-banner {
+	margin: 25px;
+}
+
+.post-share__footer-items {
+	display: flex;
+	flex-wrap: nowrap;
+	justify-content: space-between;
+
+	&::after {
+		content: '';
+		display: none;
+	}
+
+	.ellipsis-menu {
+		display: flex;
+	}
+
+	.ellipsis-menu__toggle {
+		padding-top: 0;
+		padding-bottom: 0;
+
+		.gridicon {
+			top: 4px;
+		}
+	}
+}
+
+.post-share__footer-item {
+	display: flex;
+	flex: 1 1 auto;
+	flex-direction: column;
+	overflow: hidden;
+	position: relative;
+
+	.social-logo {
+		flex: 0 0 24px;
+
+		@include breakpoint( ">660px" ) {
+			position: relative;
+			top: 1px;
+		}
+	}
+
+	.gridicons-time {
+		flex: 0 0 18px;
+		padding: 3px;
+		position: relative;
+			top: -1px;
+			left: -1px;
+
+		@include breakpoint( ">660px" ) {
+			top: 0;
+		}
+	}
+
+	@include breakpoint( ">480px" ) {
+		align-items: center;
+		flex-direction: row;
+
+		&::after {
+			@include long-content-fade( $color: $white );
+		}
+	}
+
+	@include breakpoint( ">660px" ) {
+		align-items: flex-start;
+		flex-direction: column;
+	}
+
+	@include breakpoint( ">960px" ) {
+		align-items: center;
+		flex-direction: row;
+	}
+}
+
+.post-share__handle,
+.post-share__timestamp {
+	align-items: center;
+	display: flex;
+	flex: 0 0 auto;
+	margin-right: 12px;
+
+	span {
+		padding-left: 4px;
+	}
+}
+
+.post-share__handle {
+	.social-logo.twitter {
+		color: $color-twitter;
+	}
+
+	.social-logo.facebook {
+		color: $color-facebook;
+	}
+
+	.social-logo.tumblr {
+		color: $color-tumblr;
+	}
+
+	.social-logo.google_plus {
+		color: $color-gplus;
+	}
+
+	.social-logo.linkedin {
+		color: $color-linkedin;
+	}
+
+	.social-logo.path {
+		color: $color-path;
+	}
+
+	@include breakpoint( ">480px" ) {
+		margin-right: 7px;
+	}
+
+	@include breakpoint( ">660px" ) {
+		margin-right: 12px;
+	}
+
+	@include breakpoint( ">960px" ) {
+		margin-right: 7px;
+	}
+}
+
+.post-share__timestamp {
+	span {
```

Comment:
I think we should add a custom class name here trying to avoid use the `span` selector.
Also the whole rule could be defined in the same line.